packages/fxa-settings/src/components/ThirdPartyAuth/en.ftl

```diff
@@ -0,0 +1,10 @@
+## ThirdPartyAuth component
+## This is a component that is used to display a list of third party providers (Apple, Google, etc.)
+
+# This appears when a user has the option to authenticate via third party accounts in addition to their Firefox account. 
+# Firefox account login appears on top, and third party options appear on bottom. 
+# This string appears as a separation between the two, in the following order: "Enter your password" "Or"(this string) "Continue with Google"(continue-with-google-button) / "Continue with Apple"(continue-with-apple-button)
+third-party-auth-options-or = Or
```

Comment:
Passing by note here: are you using CSS to make this uppercase? 🤔 

This string should be `OR` if you want it uppercase, as CSS is [not reliable](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#css-issues), especially when we don't control the browser.

Reply:
@flodolo Looks like the usage is:

```jsx
          <FtlMsg id="third-party-auth-options-or">
            <div className="mx-4">Or</div>
          </FtlMsg>
```

And I don't see a CSS "uppercase" class in the `div` anymore, so I'm guessing it will just be a mixed case "Or".

Reply:
And just for conversation sake, here's all the rough instances of CSS `text-transform: uppercase;` and `className="uppercase"` I could find:
```sh
git grep -n "uppercase" packages | grep -E "\b(text-transform:|className=)"

packages/fxa-admin-panel/src/components/Nav/index.tsx:21:      <h2 className="mb-3 uppercase text-sm tracking-wide font-normal text-grey-500">
packages/fxa-admin-panel/src/components/PageAccountSearch/index.tsx:245:          className="block uppercase text-sm text-grey-500"
packages/fxa-auth-server/lib/senders/emails/storybook.css:74:  text-transform: uppercase;
packages/fxa-content-server/app/styles/modules/_custom-rows.scss:198:    text-transform: uppercase;
packages/fxa-content-server/app/styles/modules/_input-row.scss:80:      text-transform: uppercase;
packages/fxa-content-server/app/styles/modules/_input-row.scss:93:      text-transform: uppercase;
packages/fxa-content-server/app/styles/modules/_settings-totp.scss:54:      text-transform: uppercase;
packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx:50:        <h2 className="text-xs text-grey-400 font-semibold uppercase">
packages/fxa-settings/src/components/Settings/UnitRowSecondaryEmail/index.tsx:201:                    className="uppercase block text-orange-600 font-bold text-xs"
packages/fxa-settings/src/components/WarningMessage/index.tsx:24:          span: <span className="text-red-600 uppercase">{warningType}</span>,
packages/fxa-settings/src/components/WarningMessage/index.tsx:28:          <span className="text-red-600 uppercase">{warningType}</span>{' '}
packages/fxa-settings/src/pages/Signin/CompleteSignin/index.tsx:69:              <span className="text-red-600 uppercase">
```

It's a bit hard to see these without context, but I'm happy to file 5 issues (one per package) if you think they merit investigation and possibly updated strings in the .ftl files.

Reply:
> It's a bit hard to see these without context, but I'm happy to file 5 issues (one per package) if you think they merit investigation and possibly updated strings in the .ftl files.

Thanks for looking into it. I'll let Bryan decide, since this is his project ;-)

Reply:
Thanks @pdehaan - I looked into the ones you found.

`fxa-admin-panel`: not localized, so not an issue
`fxa-auth-server`: this is related to storybook, so not applicable to localization
`fxa-content-server`: The only one that affects a localized string is `packages/fxa-content-server/app/styles/modules/_custom-rows.scss:198`. The content of the string is actually the same one picked up here, "Or" as relates to third party auth. If this react component is going to be replacing this, then I suspect no ticket is required. If this string will continue to live, then perhaps we might want to reinvestigate.
`fxa-settings`: I only looked at the first one `packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx:50`, but this would be a case we would probably want to avoid using uppercase transform here. So for the time being a ticket for fxa-settings seems warranted.

Reply:
> So for the time being a ticket for [fxa-settings](https://mozilla-hub.atlassian.net/browse/FXA-settings) seems warranted.

Filed! FXA-7277